src/main/java/com/napier/sem/Record.java

```diff
@@ -46,7 +47,6 @@ public Record(ResultSet result, String recordType) throws SQLException
                 region = result.getString("region");
                 population = result.getInt("population");
                 capital = result.getString("capital");
-                this.recordType = recordType;
```

Comment:
Why have you removed this? I thought the issue was that it hadn't been added to each case

Reply:
Nevermind I see it